interface_service/src/bucket.py

```diff
@@ -0,0 +1,43 @@
+"""
+Provides download and upload functionality
+For interfacing with MinIO buckets
+"""
+from minio import Minio
+from config import settings
+
+
+def authenticate():
+    """Authenticates to MinIO.
+
+    Returns:
+        Minio: The MinIO client object.
+    """
+
+    # Get credentials from env variables
+    return Minio(
+        settings["MINIO_ENDPOINT"],
```

Comment:
It's not `MINIO_ENDPOINT`, cause S3 protocol is not bound to specific implementation. Better name would be `OBJECT_STORAGE_ENDPOINT`.

Also, as I mentioned earlier, it'd be good to have all these names in separate file, in dedicated class, so that we don't have magic strings floating around the project. The properties should be also validated and have their default values configured in the file, where Dynaconf is initialized.